Scripts/Generator/MapLoader.cs

```diff
@@ -288,9 +292,10 @@ private Vector2[][] PolygonFromBitmap(bool[,] bitmap)
             for (int x = 0; x < nsize; ++x)
             {
                 if (visitedCell[x, y]) continue;
-                if (!bitmap[x, y] && bitmap[x, y + 1])
+                // want to start tracing when a cell is "off" and has an "on" cell below it
+                if (!paddedBitmap[x, y] && paddedBitmap[x, y + 1])
                 {
-                    points.Add(TracePolygon(bitmap, visitedCell, x, y));
+                    points.Add(TracePolygon(paddedBitmap, visitedCell, x, y));

```

Comment:
Want to change the variable name in `private Vector2[] TracePolygon(bool[,] bitmap, bool[,] visitedCell, int startX, int startY)` to `paddedBitmap` also?